ci-operator/config/openshift/hypershift-toolkit/openshift-hypershift-toolkit-release-4.3.yaml

```diff
@@ -26,7 +30,15 @@ images:
 - from: base
   to: hypershift-toolkit
 tests:
-- as: verify
-  commands: make verify
-  container:
-    from: src
+  - as: verify
```

Comment:
I didn't notice it before, sorry. You have wrong indentation here as well.

Reply:
updated !